src/libsystemd-network/sd-dhcp-server.c

```diff
@@ -548,13 +478,61 @@ static int server_send_ack(
                         return r;
         }
 
-        r = dhcp_server_send_packet(server, req, packet, DHCP_ACK, offset);
+        if (!is_offer) {
```

Comment:
The order in which the options are appended is now different - I assume it doesn't really matter?

Reply:
(ie: lease type and timezone were before extra/vendor specific options, now they are appended afterwards)

Reply:
It *should* not matter. But it'd be nicer to keep the original order, which was more logical. It should be enough to move this block up.